larva/src/main/java/nl/nn/adapterframework/testtool/TestTool.java

```diff
@@ -158,28 +158,33 @@ public static void runScenarios(ServletContext application, HttpServletRequest r
 		String paramAutoScroll = request.getParameter("autoscroll");
 		String paramExecute = request.getParameter("execute");
 		String paramWaitBeforeCleanUp = request.getParameter("waitbeforecleanup");
-		String paramScenarioTimeout = request.getParameter("timeout");
+		String paramGlobalTimeout = request.getParameter("timeout");
+		if(paramGlobalTimeout != null) {
+			try {
+				globalTimeout = Integer.parseInt(paramGlobalTimeout);
+			} catch(NumberFormatException e) {
+			}
+		}
 		String servletPath = request.getServletPath();
 		int i = servletPath.lastIndexOf('/');
 		String realPath = application.getRealPath(servletPath.substring(0, i));
 		String paramScenariosRootDirectory = request.getParameter("scenariosrootdirectory");
 		IbisContext ibisContext = getIbisContext(application);
 		AppConstants appConstants = getAppConstants(ibisContext);
 		runScenarios(ibisContext, appConstants, paramLogLevel,
-				paramAutoScroll, paramExecute, paramWaitBeforeCleanUp, paramScenarioTimeout,
+				paramAutoScroll, paramExecute, paramWaitBeforeCleanUp,
 				realPath, paramScenariosRootDirectory, out, silent);
 	}
 
 	public static final int ERROR_NO_SCENARIO_DIRECTORIES_FOUND=-1;
 	/**
 	 * 
-	 * @param paramScenarioTimeout 
 	 * @return negative: error condition
 	 * 		   0: all scenarios passed
 	 * 		   positive: number of scenarios that failed
 	 */
 	public static int runScenarios(IbisContext ibisContext, AppConstants appConstants, String paramLogLevel,
-			String paramAutoScroll, String paramExecute, String paramWaitBeforeCleanUp, String paramScenarioTimeout,
+			String paramAutoScroll, String paramExecute, String paramWaitBeforeCleanUp,
 			String realPath, String paramScenariosRootDirectory,
```

Comment:
I think you misunderstood me when I mentioned this should be an Integer. Right now you're updating a static field which will remember the setting across multiple sessions. I believe the idea is that you can;
- Change the default with a property
- Override the default (once) with a setting in the GUI